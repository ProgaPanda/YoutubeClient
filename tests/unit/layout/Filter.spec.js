import { shallowMount } from '@vue/test-utils';
import FilterContainer from '@/layout/Filter.layout.vue';

describe('FilterContainer', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(FilterContainer, {
      propsData: { show: true },
    });

    expect(wrapper.element).toMatchSnapshot();
  });
});
