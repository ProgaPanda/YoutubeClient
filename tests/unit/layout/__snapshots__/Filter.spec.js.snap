// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterContainer renders correctly 1`] = `
<div>
  <transition-stub
    name="slide"
  >
    <div
      class="filter-container filter-container--desktop"
    />
  </transition-stub>
   
  <div
    class="filter-container filter-container--mobile"
  />
</div>
`;
